src/test/ui/nll/user-annotations/method-ufcs-inherent-1.rs

```diff
@@ -0,0 +1,19 @@
+#![feature(nll)]
+
+// Check that substitutions given on the self type (here, `A`) carry
+// through to NLL.
+
+struct A<'a> { x: &'a u32 }
+
+impl<'a> A<'a> {
+    fn new<'b, T>(x: &'a u32, y: T) -> Self {
```

Comment:
why is there a `'b` here?

Reply:
(I see the same thing occuring in the other tests. To be honest I had thought that we didn't allow unreferenced lifetimes in generic binders ... but I guess that is just a lint, when it comes to lifetimes...?)

Reply:
copy and paste error, I think